src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -200,19 +215,51 @@ protected void install() throws InterruptedException {
 
             if (whatHappened == WhatHappened.initialized || node == null) {
                 updateDeviceGroups(whatHappened, deviceGroupTopics);
-                updateCAType(caTypeTopic);
+                updateCertificateAuthority();
             } else if (node.childOf(DEVICE_GROUPS_TOPICS)) {
                 updateDeviceGroups(whatHappened, deviceGroupTopics);
-            } else if (node.childOf(CA_TYPE_TOPIC)) {
-                if (caTypeTopic.getOnce() == null) {
+            } else if (node.childOf(CERTIFICATE_AUTHORITY_TOPIC)) {
+                if (certificateAuthorityTopic.getOnce() == null) {
                     return;
                 }
-                updateCAType(caTypeTopic);
+                updateCertificateAuthority();
             }
         });
 
     }
 
+    private void updateCertificateAuthority() {
+        String caKeyPath = Coerce.toString(config.lookup(CERTIFICATE_AUTHORITY_TOPIC, CA_PRIVATE_KEY_PATH).dflt(""));
```

Comment:
I don't think that using `dflt` here is appropriate. You can use `findOrDefault`, or just handle the null later on